scripts/pgsql/stop_places.sql

```diff
@@ -179,7 +143,7 @@ BEGIN
   ELSEIF tags->>'subway' = 'yes'
     THEN result := 'metroPlatform';
   ELSEIF tags->>'tram' = 'yes' THEN
-    IF ST_GeometryType(geom) = 'ST_Point'
+    IF geom::geometry = 'ST_Point'
```

Comment:
I feel like this will check if the content of `geom` is equal to `'ST_Point'` and not the type of `geom` as before.

Reply:
Thanks for the notice. I have changed it to `ST_GeometryType(geom::geometry)` in commit 603297a